static/src/js/schemas/uiSchemas/collections/spatialInformation.jsx

```diff
@@ -0,0 +1,1375 @@
+                            }
+                          }]
+                        },
+                        {
+                          'ui:row': [{
+                            'ui:col': {
+                              md: 12,
+                              children: ['Unit']
+                            }
+                          }]
+                        }
+                      ]
+                    }
+                  }
+                ]
+              }
+            }
+          },
+          GenericResolutions: {
+            'ui:heading-level': 'h6',
+            items: {
+              'ui:field': 'layout',
+              'ui:layout_grid': {
+                'ui:row': [
+                  {
+                    'ui:hide-header': true,
+                    'ui:col': {
+                      md: 12,
+                      children: [
+                        {
+                          'ui:row': [{
+                            'ui:col': {
+                              md: 12,
+                              children: ['XDimension']
+                            }
+                          }]
+                        },
+                        {
+                          'ui:row': [{
+                            'ui:col': {
+                              md: 12,
+                              children: ['YDimension']
+                            }
+                          }]
+                        },
+                        {
+                          'ui:row': [{
+                            'ui:col': {
+                              md: 12,
+                              children: ['Unit']
+                            }
+                          }]
+                        }
+                      ]
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        },
+        LocalCoordinateSystem: {
+          'ui:heading-level': 'h6'
+        }
+      }
+    },
+    VerticalSpatialDomains: {
+      'ui:heading-level': 'h5',
+      items: {
+        'ui:field': 'layout',
+        'ui:layout_grid': {
+          'ui:row': [
+            {
+              'ui:hide-header': true,
+              'ui:col': {
+                md: 12,
+                children: [
+                  {
+                    'ui:row': [
+                      {
+                        'ui:col': {
+                          md: 6,
+                          children: ['Type']
+                        }
+                      }
+                    ]
+                  },
+                  {
+                    'ui:row': [
+                      {
+                        'ui:col': {
+                          md: 12,
+                          children: ['Value']
+                        }
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      }
+    },
+    OrbitParameters: {
```

Comment:
This does not appear to be rendering right on my end
<img width="1086" alt="Screenshot 2024-03-11 at 12 18 16 PM" src="https://github.com/nasa/mmt/assets/135638667/23b0a7aa-f975-483f-be25-e74c2b2b614a">